### PR TITLE
Copy explorer url

### PR DESCRIPTION
**Pull request by lorenzoPrimi**
Fixes https://github.com/ooni/probe/issues/856

## Proposed Changes

  - Calling the measurements url every time opening this screen (if json file is on disk) and delete it when published. Having in this case the “status” of the publishing of the measurement in this screen
  - Showing different messages if the measurement has been published or not
  - Copies the url to clipboard

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -0,0 +1,116 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (bassosimone) [inline .idea/codeStyles/Project.xml]: Is it intended to commit this file?

File: app/src/main/java/org/openobservatory/ooniprobe/activity/MeasurementDetailActivity.java [hunk 2]
```diff
@@ -68,7 +71,7 @@
     Toolbar toolbar;
     private Measurement measurement;
     private Snackbar snackbar;
-
+    private Boolean isInExplorer;
```
Comment (bassosimone) [inline app/src/main/java/org/openobservatory/ooniprobe/activity/MeasurementDetailActivity.java]: Please keep an empty line between private variables and methods. I find it more readable.

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (lorenzoPrimi) [reply .idea/codeStyles/Project.xml]: Android studio keep adding it every time i remove it, I guess it's a file needed by the latest update of AS

Comment (bassosimone) [reply .idea/codeStyles/Project.xml]: OK

**Pull request merged**